NEOSPlus/Components/Wizards/LightSourceWizardPlus.cs

```diff
@@ -0,0 +1,262 @@
+            uiBuilder23.Button("Set Spot Angle", action9);
+
+            UIBuilder uiBuilder14 = ui;
+            uiBuilder14.Text("-------");
+
+            //Enable Lights
+            UIBuilder uiBuilder15 = ui;
+            ButtonEventHandler action4 = new ButtonEventHandler(this.Enable);
+            uiBuilder15.Button("Enable Lights", action4);
+
+            //Disable Lights
+            UIBuilder uiBuilder16 = ui;
+            ButtonEventHandler action5 = new ButtonEventHandler(this.Disable);
+            uiBuilder16.Button("Disable Lights", action5);
+
+            //Remove Lights
+            UIBuilder uiBuilder17 = ui;
+            ButtonEventHandler action6 = new ButtonEventHandler(this.Remove);
+            uiBuilder17.Button("Delete Lights", action6);
+        }
+
+        [SyncMethod]
+        private void SetShadowType(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.ShadowType.CreateUndoPoint(); l.ShadowType.Value = (ShadowType)this.TargetShadowType; }));
+
+        [SyncMethod]
+        private void MultiplyIntensity(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.Intensity.CreateUndoPoint(); l.Intensity.Value *= (float)this._intensityField.Target.ParsedValue; }));
+
+        [SyncMethod]
+        private void SetIntensity(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.Intensity.CreateUndoPoint(); l.Intensity.Value = (float)this._intensityField.Target.ParsedValue; }));
+
+        [SyncMethod]
+        private void MultiplyRange(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.Range.CreateUndoPoint(); l.Range.Value *= (float)this._rangeField.Target.ParsedValue; }));
+
+        [SyncMethod]
+        private void SetRange(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.Range.CreateUndoPoint(); l.Range.Value = (float)this._rangeField.Target.ParsedValue; }));
+
+        [SyncMethod]
+        private void SetSpotAngle(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.SpotAngle.CreateUndoPoint(); l.SpotAngle.Value = (float)this._spotAngleField.Target.ParsedValue; }));
+
+        [SyncMethod]
+        private void Remove(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => l.UndoableDestroy()));
+
+        [SyncMethod]
+        private void Disable(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.EnabledField.CreateUndoPoint(); l.Enabled = false; }));
+
+        [SyncMethod]
+        private void Enable(IButton button, ButtonEventData eventData) => this.ForeachLight((Action<Light>)(l => { l.EnabledField.CreateUndoPoint(); l.Enabled = true; }));
+
+        /// <summary>
+        /// Easily loop oiver all lights selected and apply a process to them
```

Comment:
Easily loop _over_ all lights selected and apply a process to them 🙃

Reply:
Whoops, fixed in the latest commit